scripts/nix-profile.sh.in

```diff
@@ -34,4 +81,7 @@ if [ -n "$HOME" ]; then
     elif [ -e "$NIX_LINK/etc/ca-bundle.crt" ]; then # old cacert in Nix profile
         export SSL_CERT_FILE="$NIX_LINK/etc/ca-bundle.crt"
     fi
+
+    export PATH="$NIX_LINK/bin:$NIX_LINK/sbin:$__savedpath"
```

Comment:
the quotes don't hurt anything, do they? It's never been totally clear to me exactly when you can safely omit quotes in shell variable assignments.